bindings/netstandard/ElectionGuard/ElectionGuard.Encryption.Tests/ElectionGuard.Encryption.Tests.csproj

```diff
@@ -18,19 +18,20 @@
   </ItemGroup>
 
   <PropertyGroup Label="Library Paths">
+    <MsvcTarget>Release</MsvcTarget>

```

Comment:
The unit tests were failing in the Visual Studio test runner because the csproj files use `$(Configuration)` in the file path to get electionguard.dll.  Configuration is Debug in the IDE, as one would expect when running tests for compilation speed and stack trace accuracy.  However, `make test-msvc` generates electionguard into a Release folder by default.  This change ^ should make life easier for new users to this codebase.

Reply:
I believe this exists for non window systems. I believe locking this command this way will break this line.
https://github.com/microsoft/electionguard-cpp/blob/838472b1521a1eb423ad42d23eea3e4634883b2c/Makefile#L180

Reply:
In addition, this feels like a separate issue.

Reply:
I believe this exists for non window systems. I believe locking this command this way will break this line for changing the configuration on other systems. 

https://github.com/microsoft/electionguard-cpp/blob/838472b1521a1eb423ad42d23eea3e4634883b2c/Makefile#L180

Also, feels like separate issue.